### ISSUE TITLE
Memcache addon is not available at Heroku please change description.

### ISSUE THREAD

**Issue by Koziolek** [nhurion/hello-vaadin-heroku#3]
At this moment Heroku command `heroku addons:add memcache` return 

`!    AddonPlan not found.`

**Comment by nhurion** [comment]
Yes, heroku stopped the memcache addon and replaced it by another one.

This project need some upgrades, I'll try to work on it this week or next week.

**Comment by ghost** [comment]
Any upgrades on this issue? Is it ok to use `heroku addons:add memcachedcloud`?
